### PR TITLE
lite-xl: 2.1.1 -> 2.1.3

### PR DESCRIPTION
**Pull request by sefidel**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by raphaelr — APPROVED** [review]
Release notes: https://github.com/lite-xl/lite-xl/releases/tag/v2.1.2

**Comment by SuperSandro2000** [comment]
maybe the dependencies for darwin are now cached

@ofborg build lite-xl

File: pkgs/applications/editors/lite-xl/default.nix [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
   pname = "lite-xl";
-  version = "2.1.1";
+  version = "2.1.2";
 
   src = fetchFromGitHub {
     owner = "lite-xl";
     repo = "lite-xl";
     rev = "v${version}";
-    sha256 = "sha256-tZ9bCazs4ygNl5RKFNUtxboaMcG8a7mIz2FuiExX1d4=";
+    sha256 = "sha256-kl3vgBP+ExgZfdyOlqAEhUdR+kG8dAjyvVSm2I4e9yg=";
```
Comment (kirillrdy) [inline pkgs/applications/editors/lite-xl/default.nix:24]:
```suggestion
    hash = "sha256-kl3vgBP+ExgZfdyOlqAEhUdR+kG8dAjyvVSm2I4e9yg=";
```

Comment (sefidel) [reply pkgs/applications/editors/lite-xl/default.nix]: Thanks, resolved in e17bbcca2e1bf27083433f01911a8d304dcd724e!

**Comment by sefidel** [comment]
`v2.1.3` was released last week, I've bumped the version.

**Comment by sefidel** [comment]
Note: needs testing on darwin.

**Comment by kirillrdy** [comment]
on darwin, one of indirect dependencies, `agg` fails to build

**Comment by kirillrdy** [comment]
https://github.com/NixOS/nixpkgs/pull/287376 fixes agg on darwin ... but lite-xl builds/runs without agg, some should check history

**Pull request merged**